setup.cfg

```diff
@@ -20,6 +20,10 @@ package_dir =
     = src
 packages = find:
 python_requires = >=3.6
+install_requires =
```

Comment:
This is what makes sure we satisfy install dependencies. The requirements.txt file is used purely for development.